src/core/shape_inference/include/embeddingbag_packed_shape_inference.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/validation_util.hpp"
+#include "openvino/op/util/embeddingbag_packed_base.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+
+template <class TShape>
+std::vector<TShape> shape_infer(const ov::op::util::EmbeddingBagPackedBase* op,
+                                const std::vector<TShape>& input_shapes) {
+    const auto input_size = input_shapes.size();
+    NODE_VALIDATION_CHECK(op, input_size == 2 || input_size == 3);
+
+    static constexpr int EMB_TABLE = 0;
```

Comment:
```suggestion
    constexpr size_t EMB_TABLE = 0;
```
There is not much benefit from (compiler handle optimization) and if possible make it unsigned.

Reply:
Ok, done